microsoft-edge/devtools-guide-chromium/whats-new/whats-new.md

```diff
@@ -23,22 +23,4 @@ ms.date: 05/16/2022
 * [What's New in DevTools (Microsoft Edge 102)](2022/05/devtools-102.md)
 * [What's New in DevTools (Microsoft Edge 101)](2022/04/devtools-101.md)
 * [What's New in DevTools (Microsoft Edge 100)](2022/03/devtools-100.md)
-* [What's New in DevTools (Microsoft Edge 99)](2022/03/devtools.md)
-* [What's New in DevTools (Microsoft Edge 98)](2022/02/devtools.md)
-* [What's New in DevTools (Microsoft Edge 97)](2022/01/devtools.md)
-* [What's New in DevTools (Microsoft Edge 96)](2021/11/devtools.md)
-* [What's New in DevTools (Microsoft Edge 95)](2021/10/devtools.md)
-* [What's New in DevTools (Microsoft Edge 94)](2021/09/devtools.md)
-* [What's New in DevTools (Microsoft Edge 93)](2021/07/devtools.md)
-* [What's New in DevTools (Microsoft Edge 92)](2021/05/devtools.md)
-* [What's New in DevTools (Microsoft Edge 91)](2021/04/devtools.md)
-* [What's New in DevTools (Microsoft Edge 90)](2021/02/devtools.md)
-* [What's New in DevTools (Microsoft Edge 89)](2021/01/devtools.md)
-* [What's New in DevTools (Microsoft Edge 88)](2020/11/devtools.md)
-* [What's New in DevTools (Microsoft Edge 87)](2020/10/devtools.md)
-* [What's New in DevTools (Microsoft Edge 86)](2020/08/devtools.md)
-* [What's New in DevTools (Microsoft Edge 85)](2020/06/devtools.md)
-* [What's New in DevTools (Microsoft Edge 84)](2020/05/devtools.md)
-* [What's New in DevTools (Microsoft Edge 83)](2020/03/devtools.md)
-* [What's New in DevTools (Microsoft Edge 81)](2020/01/devtools.md)
-* [What's New in DevTools (Microsoft Edge 80)](2019/12/devtools.md)
+* [What's New in DevTools Archive](whats-new-archive.md)
```

Comment:
```suggestion
* [What's New in DevTools Archive](./whats-new-archive.md)
```
Adding this markup isn't gratuitous complexification, it's to make autocomplete work in VS Code, and b/c the tools do this when Alt+M -> link > add.

Reply:
accepted